config/database.go

```diff
@@ -293,7 +293,13 @@ func (ds *DatabaseStore) RemoveFile(name string) error {
 
 // String returns the path to the database backing the config, masking the password.
 func (ds *DatabaseStore) String() string {
-	u, _ := url.Parse(ds.originalDsn)
+	tcpStripper := regexp.MustCompile(`@tcp\((.*)\)`)
```

Comment:
Can we declare this as a var at the package level to avoid recompiling every the regular expression every time? (The compiler might be smart enough, but I think it's clearer to hoist this out.)

Reply:
This is one of the cases where I suppose the compiler is smarter then me. I'm trying to avoid global variables when ever I can and this seams an easy one not use a global variable. 2/5

Still, applied your suggestion so we can this merged soon.